src/setup.rs

```diff
@@ -52,6 +56,19 @@ pub async fn setup(config: &Configuration, tracker: Arc<tracker::Tracker>) -> Ve
         jobs.push(tracker_api::start_job(&config.http_api, tracker.clone()).await);
     }
 
+    // Start HTTP APIs server (multiple API versions)
+    if config.http_api.enabled {
+        // Temporarily running the new API in the 1313 port
+        let bind_address = config.http_api.bind_address.clone();
+        let mut bind_socket: SocketAddr = bind_address.parse().unwrap();
```

Comment:
prefer expects over unwrap?

Reply:
I've changed it, but we have a lot of "unwraps". For example like this:

```
InfoHash::from_str(&hash).unwrap();
```

I think it would not be very pleasant to write a message for all of them:

```
InfoHash::from_str(&hash).expect("string should be a valid infohash");
```

I have to think about it, but I would say it makes sense when the case is unique.

Maybe the problem is we are duplicating code, and we should add a function like this:

```rust
fn parse_socket_addr_or_fail(bind_address: &str) -> SocketAddr {
    bind_address.parse().expect("the bind address string should contain a valid socket address like this 127.0.1.1:8080")
}
```